validation.py

```diff
@@ -23,16 +23,23 @@ def validate_size(self, size):
         if not self.is_valid_integer(size):
             return ValidationResult(False, "Eek! That's not even a number! ")
 
-        size = int(size)
-
-        if not self.is_in_range(size, range(3, 6)):
+        if not self.is_in_range(int(size), range(3, 6)):
             return ValidationResult(False, "Whoa friend! This is outta bounds! ")
 
-        if not self.is_odd(size):
+        if not self.is_odd(int(size)):
             return ValidationResult(False, "Ummm. This isn't odd friend!")
 
         return ValidationResult(True, "")
 
+    def validate_choice(self, choice):
```

Comment:
Since this is validating a menu choice, we can rename it to be closer to what it is checking, validate_menu_choice.